build/build.sh

```diff
@@ -82,4 +82,11 @@ if [ "${BUILD_DOCX:-}" = "true" ]; then
     --defaults="$PANDOC_DEFAULTS_DIR/docx.yaml"
 fi
 
+# Spellcheck
+if [ "${SPELLCHECK:-}" = "true" ]; then
+  export ASPELL_CONF="add-extra-dicts $(pwd)/build/assets/custom-dictionary.txt"
+  pandoc --lua-filter spellcheck.lua output/manuscript.md | sort | uniq > output/spelling-errors.txt
```

Comment:
Can we change 
```bash 
pandoc --lua-filter spellcheck.lua output/manuscript.md | sort | uniq > output/spelling-errors.txt
``` 
to the following line:
```bash
pandoc --lua-filter spellcheck.lua output/manuscript.md | uniq | while read word; do grep -on "\<$word\>" output/manuscript.md; done | sort -h > output/spelling-errors.txt
```
^ This adds line numbers to each misspelled word. Would be useful to identify the exact spot of the error.

Reply:
Good idea @danich1.  I tested that in 14215d36694d960475f30c6eab334237a8e43ec0 and the output is below.

Is this line numbering useful enough? The line numbers correspond to `output.md`, but authors will have to edit the original `*.md` files in the `content` subdirectory to correct the spelling errors.

Reply:
Good point. This fix appends file names at the beginning of each "misspelled" word:
```bash
pandoc --lua-filter spellcheck.lua output/manuscript.md | uniq | while read word; do grep -on "\<$word\>" content/*md; done | sort -h -t ":" -k 1,2 > output/spelling-errors.txt
```
^ Assuming the desire is to only spell check md files.

Reply:
Thanks @danich1.  I added that in a63502f but the CI build failed.  I like this solution, but I don't think I'll be able to debug locally until tomorrow.